cli/src/commands/prev.rs

```diff
@@ -20,35 +20,35 @@ use crate::cli_util::{short_commit_hash, CommandHelper};
 use crate::command_error::{user_error, CommandError};
 use crate::commands::next::choose_commit;
 use crate::ui::Ui;
-
-/// Move the working-copy commit to the parent revision
+/// Change the working copy revision relative to the parent revision
 ///
+/// The command creates a new empty working copy revision that is the child of
+/// an ancestor OFFSET revisions behind the parent of the current working copy.
 ///
-/// The command moves you to the parent in a linear fashion.
+/// For example, when OFFSET is 1:
```

Comment:
I dislike getting screamed at in documentation, so can you lowercase both offsets here?

Reply:
Haha yes. I just capitalized it to match how the Clap generated documentation does it. I'll fix and mail a patch for `jj parallelize` for consistency.

Reply:
Actually, maybe `OFFSET` is ok in the previous paragraph where it's clearly referring to the argument name? Should I change that too or just this one?

Reply:
please do both, it really is unpleasant. You can use backticks or quotes to imply the argument.

Reply:
Ok done!